frontend/src/routes/train.jsx

```diff
@@ -0,0 +1,115 @@
+import { createMemo } from "solid-js";
+import { styled } from "solid-styled-components";
+import { createFormGroup, createFormControl } from "solid-forms";
+import Button from "../components/form/button";
+import EmailInput from "../components/form/email-input";
+import HashInput from "../components/form/hash-input";
+import FileInput from "../components/form/file-input";
+import Slider from "../components/slider";
+import { validators } from "../helpers/validators";
+import { getFilesUrls } from "../helpers/getFilesUrl";
+
+const Train = () => {
+  const { controls } = createFormGroup({
+    email: createFormControl("", {
+      validators: validators.email,
+    }),
+    files: createFormControl([], {
+      validators: validators.files,
+    }),
+    hash: createFormControl("", { readonly: true, disabled: true }),
+  });
+
+  const imageList = createMemo(() => getFilesUrls(controls.files.value));
+
+  const handleFormSubmit = async () => {
+    if (controls.email.isValid && controls.files.isValid) {
+      // Uploading the files using the fetch API to the server (temporarily with json for static)
+      const formData = new FormData();
+      const files = controls.files.value;
+      for (let i = 0; i < files.length; i++) {
+        const currFile = files[i];
+        formData.append(`files`, currFile, currFile.name);
+      }
+      const res = await fetch(
+        `${import.meta.env.VITE_API_URL}/scene-representation`,
+        {
+          method: "POST",
+          body: { files: formData, email: controls.email.value },
+        }
+      );
+      const hash = await res.json();
```

Comment:
Please, add examples of using `fetch` in solid-js

Reply:
https://stackoverflow.com/questions/74589998/rendering-remote-data-using-the-fetch-api-in-solidjs
https://stackblitz.com/edit/vitejs-vite-nfgcpt?file=src%2FApp.jsx

we can use:
  const [hash] = createResource(controls, sceneRepresentation);
instead of:
  const hash = await sceneRepresentation(controls);

but i dont see any sense of it